package.json

```diff
@@ -40,8 +35,18 @@
     "eslint-config-next": "^11.0.1",
     "husky": "^6.0.0",
     "jest": "^27.0.5",
+    "lint-staged": ">=10",
```

Comment:
I'm curious, why we're open to have major update here?

Reply:
good catch. the `npx mrm lint-staged` command changes the installed version